### PR TITLE
fix: Remove historic countries that do not exist today from country list

### LINKED ISSUE #5604: Country List in Forms show historic countries that do not exist today
Please update the country list used in forms to only show countries that exist today.

Part of the issue is, that you research the list of countries that exist today or find a library that is up to date.

Screenshot from 2020-11-13 13-25-13

### PR DESCRIPTION
**Pull request by sachinchauhan2889**
Fixes #5604 

#### Short description of what this resolves:
remove all old countries.

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [X] My branch is up-to-date with the Upstream `development` branch.
- [X] The acceptance, integration, unit tests and linter pass locally with my changes 
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/utils/dictionary/demography.ts [hunk 1]
```diff
@@ -251,22 +183,10 @@ export const countries = [
...
     name : 'Cyprus',
     code : 'CY'
   },
-  {
-    name : 'Czech Republic',
-    code : 'CZ'
-  },
   {
     name : 'German Democratic Republic',
     code : 'DD'
```
Comment (mariobehling) [inline app/utils/dictionary/demography.ts:192]: No longer in existence.

File: app/utils/dictionary/demography.ts [hunk 2]
```diff
@@ -447,14 +307,6 @@ export const countries = [
     name : 'Guyana',
     code : 'GY'
   },
-  {
-    name : 'Hong Kong',
-    code : 'HK'
-  },
```
Comment (mariobehling) [inline app/utils/dictionary/demography.ts:453]: Please keep Hong Kong.

File: app/utils/dictionary/demography.ts [hunk 3]
```diff
@@ -1035,10 +727,6 @@ export const countries = [
     name : 'Tuvalu',
     code : 'TV'
   },
-  {
-    name : 'Taiwan',
-    code : 'TW'
-  },
```
Comment (mariobehling) [inline app/utils/dictionary/demography.ts:1041]: Taiwan is an existing territory even though China does not recognize it. Where did you get the info for that list? I am not an expert on all countries but it seems not all data is correct here.

File: app/utils/dictionary/demography.ts [hunk 4]
```diff
@@ -1019,10 +715,6 @@ export const countries = [
     name : 'Tonga',
     code : 'TO'
   },
-  {
-    name : 'East Timor',
-    code : 'TP'
-  },
```
Comment (mariobehling) [inline app/utils/dictionary/demography.ts:1025]: Please check. I think it still exists.

File: app/utils/dictionary/demography.ts [hunk 5]
```diff
@@ -1047,14 +735,6 @@ export const countries = [
     name : 'Ukraine',
     code : 'UA'
   },
-  {
-    name : 'Uganda',
-    code : 'UG'
-  },
```
Comment (mariobehling) [inline app/utils/dictionary/demography.ts:1053]: Why no more Uganda?

File: app/utils/dictionary/demography.ts [hunk 6]
```diff
@@ -1063,14 +743,6 @@ export const countries = [
     name : 'Uruguay',
     code : 'UY'
   },
-  {
-    name : 'Uzbekistan',
-    code : 'UZ'
-  },
```
Comment (mariobehling) [inline app/utils/dictionary/demography.ts:1069]: still in existence

**Review by mariobehling — CHANGES_REQUESTED** [review]
These changes do not seem to be entirely correct. Please do more research about this information and also update if a country name or region name has changed. Also point to where you found the information. So we can understand it. Thanks.

**Comment by sachinchauhan2889** [comment]
> These changes do not seem to be entirely correct. Please do more research about this information and also update if a country name or region name has changed. Also point to where you found the information. So we can understand it. Thanks.

ok sir, i will check every country one by one

**Comment by MeghalBisht** [comment]
> > These changes do not seem to be entirely correct. Please do more research about this information and also update if a country name or region name has changed. Also point to where you found the information. So we can understand it. Thanks.
> 
> ok sir, i will check every country one by one

Hey, are you still working on it?

**Comment by sachinchauhan2889** [comment]
> Hey, are you still working on it?

yes

**Review by MeghalBisht — COMMENTED** [review]
You can use this resource. It has all the countries recognized by the UN(2020).
https://www.worldometers.info/geography/countries-of-the-world/

**Comment by sachinchauhan2889** [comment]
> You can use this resource. It has all the countries recognized by the UN(2020).
> https://www.worldometers.info/geography/countries-of-the-world/

@MeghalBisht thanks.

**Comment by iamareebjamal** [comment]
https://github.com/annexare/Countries/blob/master/dist/countries.json

**Comment by sachinchauhan2889** [comment]
> https://github.com/annexare/Countries/blob/master/dist/countries.json
> 
> ```js
> const countries = await (await fetch('https://raw.githubusercontent.com/annexare/Countries/master/dist/countries.json')).json()
> Object.entries(countries).map(([code, { name }]) => ({ code, name }))
> ```

**Comment by sachinchauhan2889** [comment]
> https://github.com/annexare/Countries/blob/master/dist/countries.json
> 
> ```js
> const countries = await (await fetch('https://raw.githubusercontent.com/annexare/Countries/master/dist/countries.json')).json()
> Object.entries(countries).map(([code, { name }]) => ({ code, name }))
> ```

@iamareebjamal sir, I tried this.
But _top level await_ is not working. 
getting error ---> _top-level 'await' expressions are only allowed when the 'module' option is set to 'esnext' or 'system', and the 'target' option is set to 'es2017' or higher._ 
I also used await in async function but error remain there.

Thats why I have totally matched counries in our demography,ts with your provided list. i have checked each and every country one by one and correct them and remove them.

**Comment by iamareebjamal** [comment]
I meant run it in your browser console and copy it here, but you seemed to have done the same already

**Comment by sachinchauhan2889** [comment]
> I meant run it in your browser console and copy it here, but you seemed to have done the same already

@iamareebjamal  yes, sir.

**Pull request merged**